src/python/pants/option/global_options.py

```diff
@@ -125,6 +125,12 @@ def register_bootstrap_options(cls, register):
     register('-q', '--quiet', type=bool, recursive=True, daemon=False,
              help='Squelches most console output. NOTE: Some tasks default to behaving quietly: '
                   'inverting this option supports making them noisier than they would be otherwise.')
+
+    register('--loop', type=bool, daemon=True,
```

Comment:
Probably making it a hard error to specify this flag if not `--v2 --no-v1` :)

Reply:
Yea, agreed. Will add.